core/src/test/java/org/apache/calcite/test/RelMetadataTest.java

```diff
@@ -562,6 +562,17 @@ private void checkTwoColumnOrigin(
         true, metadataConfig);
   }
 
+  @ParameterizedTest
+  @MethodSource("getArguments") void testColumnOriginsSnapshot(
+      final MetadataConfig metadataConfig) {
+    checkSingleColumnOrigin(
+        "select productid from products_temporal "
```

Comment:
Please reformat the SQL use line breaking

Reply:
Done